README.md

```diff
@@ -1,169 +1,30 @@
-  // remote attachment. On the recipient side, clients can verify that the
-  // encrypted encoded payload they've downloaded matches what was uploaded.
-  // This is to prevent tampering with the content once it's been uploaded.
-  contentDigest: encryptedAttachment.digest,
-
-  // These are the encryption keys that will be used by the recipient to
-  // decrypt the remote payload
-  salt: encryptedAttachment.salt,
-  nonce: encryptedAttachment.nonce,
-  secret: encryptedAttachment.secret,
-
-  // For now, all remote attachments MUST be fetchable via HTTPS GET requests.
-  // We're investigating IPFS here among other options.
-  scheme: "https://",
-
-  // These fields are used by clients to display some information about
-  // the remote attachment before it is downloaded and decrypted.
-  filename: attachment.filename,
-  contentLength: attachment.data.byteLength,
-};
-```
-
-### Send a remote attachment
-
-Now that you have a remote attachment, you can send it:
-
-```tsx
-await conversation.messages.send(remoteAttachment, {
-  contentType: ContentTypeRemoteAttachment,
-    contentFallback: "a screenshot of 1MB of text",
-})
-```
-
-Note that we’re using `contentFallback` to enable clients that don't support these content types to still display something. For cases where clients *do* support these types, they can use the content fallback as alt text for accessibility purposes.
-
-### Receive a remote attachment
-
-Now that you can send a remote attachment, you need a way to receive a remote attachment. For example:
-
-```tsx
-// Assume `loadLastMessage` is a thing you have
-const message: DecodedMessage = await loadLastMessage()
-
-if (!message.contentType.sameAs(ContentTypeRemoteAttachment)) {
-    // We do not have a remote attachment. A topic for another blog post.
-    return
-}
-
-// We've got a remote attachment.
-const remoteAttachment: RemoteAttachment = message.content
-```
-
-### Download, decrypt, and decode the attachment
-
-Now that you can receive a remote attachment, you need to download, decrypt, and decode it so your app can display it. For example:
-
-```tsx
-const attachment: Attachment = await RemoteAttachmentCodec.load(
-    remoteAttachment,
-    client, // <- Your XMTP Client instance
-)
-```
-
-You now have the original attachment:
-
-```tsx
-attachment.filename // => "screenshot.png"
-attachment.mimeType // => "image/png",
-attachment.data // => [the PNG data]
-```
-
-### Display the attachment
-
-Display the attachment in your app as you please. For example, you can display it as an image:
-
-```tsx
-const objectURL = URL.createObjectURL(
-    new Blob([Buffer.from(attachment.data)], {
-    type: attachment.mimeType,
-  }),
-);
-
-const img = document.createElement('img')
-img.src = objectURL
-img.title = attachment.filename
-```
-
-To learn more, see [Some new content types](https://xmtp.org/blog/attachments-and-remote-attachments).
+- `yarn build`: Build all packages
```

Comment:
I know this might be obvious to devs - but should we call out `yarn install` as the first step? Feel free to ignore if you think it'll be more confusing than helpful!

```suggestion
- `yarn install`: Install all packages
- `yarn build`: Build all packages
```

Reply:
installing dependencies, whether in `yarn`, `npm`, or `pnpm` is implicit and part of the normal dev worflow for all web projects using dependencies. i don't think need to call it out.